packages/connect/src/types/device.ts

```diff
@@ -29,6 +29,15 @@ export enum FirmwareType {
     Regular = 'regular',
 }
 
+export type DeviceState = {
+    // sessionId:
+    // - T1B1 < 1.9.0 passphrase string received from UI
+    // - T2T1 < 2.3.0 result of Deprecated_PassphraseStateRequest message
```

Comment:
@Hannsek are we allowed to stop supporting these firmwares? In the way that connect would reject everything except firmware update?

Reply:
Yes, I'll just write it to some slack channel.

Reply:
these fws are already deprecated and their handling code has been removed. probably source of conflicts in this PR